code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm

```diff
@@ -44,6 +44,8 @@
 	return type
 
 /obj/machinery/cooker/deepfryer/attackby(obj/item/I, mob/user, params)
+	if(istype(I, /obj/item/grab))
+		return special_attack_grab(I, user)
```

Comment:
Is it necessary to duplicate this instead of letting the parent run this?

Reply:
Nope, good call.